backend/tests/controllers/test_executions/test_test_executions.py

```diff
@@ -252,7 +253,7 @@ def test_report_test_execution_data(db_session: Session, test_client: TestClient
     assert test_execution.test_results[1].status == TestResultStatus.SKIPPED
 
 
-def test_updates_test_execution(db_session: Session, test_client: TestClient):
+def _prepare_test_execution_object(db_session: Session) -> TestExecution:
```

Comment:
I have no strong opinions on this. It's fine that this is a helper function but it could also be a fixture